synapse/lib/stormtypes.py

```diff
@@ -5783,14 +5783,26 @@ class View(Prim):
         {'name': 'parent', 'desc': 'The parent View. Will be ``$lib.null`` if the view is not a fork.', 'type': 'str'},
         {'name': 'triggers', 'desc': 'The ``storm:trigger`` objects associated with the ``storm:view``.',
          'type': 'list', },
-        {'name': 'set', 'desc': 'Set a arbitrary value in the View definition.',
+        {'name': 'set', 'desc': '''
```

Comment:
note: will need to test rendering this to make sure it looks clean.

Reply:
Is there a known-good way?

Reply:
Build the docs locally and then open them in a browser :D

I'm happy to check them.